src/phases/evolution-phase.ts

```diff
@@ -189,75 +189,75 @@ export class EvolutionPhase extends Phase {
                             }
                           });
 
-                          SoundFade.fadeOut(this.scene, evolutionBgm, 100);
+                          SoundFade.fadeOut(evolutionBgm, 100);
```

Comment:
```suggestion
                          SoundFade.fadeOut(gScene, evolutionBgm, 100);
```
The issue seems to be here. Some of the calls to SoundFade.fadeOut kept the scene, but for others it got removed like here and in Pokemon.ts, hatch phase. But it's a Phaser thing and still needs the scene
I can't test too much but changing this file fixed the evolution test. And leaving the hatch scene as it is crashes the game during hatching, so I think it's safe to say that is the problem

Reply:
Ah, right, for some reason I was thinking it only needed to be changed in one place since most things still worked after that change.